### PR TITLE
[2.13.x] DDF-8324: Fix stability issues in itests

### PR DESCRIPTION
**Pull request by figliold**
#### What does this PR do?
Fix stability issues in itests
#### Who is reviewing it? 
@ethantmanns 
@tyler30clemens 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@tbatie
#### How should this be tested?
CI build

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ethantmanns** [comment]
build now

**Comment by ethantmanns** [comment]
I ran these tests locally and didn't see any issues. I'm not sure why there were failures on the last CI build, but I'm going to attempt another build as I look into those.

build now

**Comment by figliold** [comment]
build now

Increased generic timeout as tests were timing out running catalog:removeall

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -139,9 +139,9 @@
 
   private static final File UNPACK_DIRECTORY = new File("target/exam");
 
-  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(10);
+  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(20);
```
Comment (clockard) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: I thought 10 minutes was already way to much time. I think we need a different approach. What if we change REMOVE_ALL to `catalog:removeall -f` which will leave the geoname metacards alone. If some of the tests need to remove other non resource metacards then they could be updated to do something like `catalog:remove --cql "\\\"metacard-tags\\\" =  'mytype'"`. This way we only remove the small number of metacards created by the tests.

**Comment by clockard** [comment]
Bypassed the sonarqube check on this one because only test classes were changed.

**Pull request merged**